Schutzfile

```diff
@@ -2,7 +2,7 @@
   "fedora-37": {
     "dependencies": {
       "osbuild": {
-        "commit": "1fbd9d975f15b845b9a9519097cd307253a64f1e"
+        "commit": "06072d5d899a995cd1e41b8865c7ac9e21901570"
```

Comment:
This commit won't work, since the post-release bump commit skips CI and there are no RPMs built as a result. You need to use the commit with the release tag (the parent of this one).

Reply:
Ack.

Reply:
Updated to refer to the commit before-the-bump.